torrent/choker.go

```diff
@@ -80,7 +80,8 @@ func (ccp *ClassicChokePolicy) Choke(chokers []Choker) (unchokeCount int, err er
 	if optimistIndex < 0 {
 		candidateCount := len(chokers) - OPTIMISTIC_UNCHOKE_INDEX
 		if candidateCount > 0 {
-			candidate := OPTIMISTIC_UNCHOKE_INDEX + rand.Intn(candidateCount)
+			r := rand.New(rand.NewSource(time.Now().UnixNano()))
+			candidate := OPTIMISTIC_UNCHOKE_INDEX + r.Intn(candidateCount)
```

Comment:
Doesn't really matter if we use the default seed. It's random enough for this use.